packages/dashboard-frontend/src/store/FactoryResolver/index.ts

```diff
@@ -142,10 +142,34 @@ export const actionCreators: ActionCreators = {
             error_code: factoryParams?.errorCode,
           })
         : undefined;
+      const isDevfileRegistryLocation = (location: string): boolean => {
+        const devfileRegistries = [
+          `${window.location.protocol}//${window.location.host}${DEFAULT_REGISTRY}`,
+        ];
+        if (state.dwServerConfig.config.devfileRegistryURL) {
+          devfileRegistries.push(state.dwServerConfig.config.devfileRegistryURL);
+        }
+        const externalDevfileRegistries =
+          state.dwServerConfig.config.devfileRegistry.externalDevfileRegistries.map(
+            externalDevfileRegistriy => externalDevfileRegistriy.url,
+          );
+        if (externalDevfileRegistries.length) {
+          devfileRegistries.push(...externalDevfileRegistries);
+        }
+        let isRegistryLocation = false;
+        devfileRegistries.every(registry => {
+          if (location.startsWith(registry)) {
+            isRegistryLocation = true;
+            return false;
+          }
+          return true;
+        });
```

Comment:
```suggestion
        const isRegistryLocation = devfileRegistries.some(registry => location.startsWith(registry));
```

Reply:
Ok. I have fixed it.

Reply:
https://github.com/eclipse-che/che-dashboard/pull/867/commits/1373031317ecbc57f89dfb0a527671ade3e09e63